modules/local/annotation/full_annotation/main.nf

```diff
@@ -30,7 +27,12 @@ process ANNOTATION {
     grep -vf $biotypes $gtf > filt.gtf
     mv $bed circs.bed
 
-    annotate_outputs.sh $exon_boundary &> ${prefix}.log
+    mkdir -p bed12
+
+    parallel -j $task.cpus -a circs.bed annotate_outputs.sh $exon_boundary {}
```

Comment:
Since order doesn't matter, you could use `parallel -u` to speed up output. I went with a split > parallel > pool approach in [my fork](https://github.com/rreggiar/circrna/commit/904104a647fc82fdf93724f452231fe2df9cb710#diff-f53b1433313394a305660960d73aec029377fdec571a89a3359e7b4a4fb14fa6R41-R48) and feel like the implementation in this commit can be optimized, but as long as this works for now we can worry about that later.